waspc/src/Wasp/Generator/ServerGenerator.hs

```diff
@@ -104,10 +104,11 @@ genPackageJson wasp waspDeps waspDevDeps =
             "devDepsChunk" .= npmDevDepsToPackageJsonEntry waspDevDeps,
             "nodeVersion" .= nodeVersionAsText,
             "startProductionScript"
-              .= concat
-                [ if not (null $ Wasp.getPSLEntities wasp) then "npm run db-migrate-prod && " else "",
-                  "NODE_ENV=production node ./src/server.js"
-                ]
+              .= if not (null $ Wasp.getPSLEntities wasp)
```

Comment:
Can we format this nicer, indent then/else more, and put parenthesses around `if-then-else` so it is clear `++` applies to the whole `if-then-else`?

Reply:
We could but it would then be a battle between hlint (that will warn of redundant parenthesis) and ormolu (that will wreck a nicer hand-crafted for legibility layout).

Reply:
Blah, I do value these tools but sometimes they are a PITA. Ok, let's leave it like this then!